### DISCUSSION TITLE
How to generate api from comment in my defs?

### DISCUSSION THREAD

**Discussion by ghost** [fastapi/fastapi#7966] [Questions] [answered]
**How to generate api from comment in my defs (response and other) and why i see empty Validation Error?**

How to i can edit this?

**Additional context**
![DeepinScreenshot_выберите-область_20191027214314](https://user-images.githubusercontent.com/1178208/67638089-b3419980-f8e9-11e9-9956-af6c5fb74a60.png)
![DeepinScreenshot_выберите-область_20191027214234](https://user-images.githubusercontent.com/1178208/67638091-b3419980-f8e9-11e9-8024-2069db95e056.png)

**Comment by prostomarkeloff** [comment, marked as answer]
You should write this not in comment, in decorator declaration.

Example: https://fastapi.tiangolo.com/tutorial/additional-responses/

**Comment by tiangolo** [comment]
Thanks for the help here @prostomarkeloff ! :clap: :bow: 

Thanks for coming back and closing the issue @foozzi  :+1:
